randombytes.c

```diff
@@ -38,12 +38,28 @@
 # endif
 #endif
 
+#ifdef __EMSCRIPTEN__
+#include <stdlib.h>
+#include <emscripten.h>
+int randombytes_js_randombytes_nodejs(void *buf, size_t n) {
+	size_t c;
+	char *bytes = (char*) EM_ASM_INT({
+			const crypto = require('crypto');
+			var out = _malloc($0);
+			writeArrayToMemory(crypto.randomBytes($0), out);
+			return out;
+		}, n);
+	for(c=0;c<n;c++)
+		((char*)buf)[c] = bytes[c];
+	free(bytes);
+	return 0;
+}
+#endif
 
 #if defined(_WIN32)
 static int randombytes_win32_randombytes(void* buf, const size_t n)
 {
 	HCRYPTPROV ctx;
-	ULONG i;
```

Comment:
slipped in, this silences a warning on win builds. can rebase it out if you prefer.

Reply:
I hadn't seen this one yet, thanks!